wiki/Brand_identity_guidelines/en.md

```diff
@@ -2,39 +2,69 @@
 
 The brand identity guidelines is a set of standards for branding osu!-related projects. All use does not require approval.
 
-[Download the design kit here.](https://drive.google.com/file/d/1TmUot5nu49p71icz4u3G68njLAQOeQrG/view?usp=sharing) The linked design kit contains some outdated assets. Also note that the included PDF is outdated and this page should be used instead.
+## Assets
+
+*Note: Some parts of the design are still work in progress, unless stated otherwise.*\
+*See also: [osu-resources — Assets used by osu!](https://github.com/ppy/osu-resources)*
+
+The [osu! Figma library](https://www.figma.com/file/6m10GiGEncVFWmgOoSyakH/osu!-Figma-Library) contains design assets such as the osu! logo, mod icons, colour palettes, and many more. To obtain them, log into Figma, mark objects for download using rectangular selection, and click the `Export` tab in the top right corner.
+
+The osu! website design is licensed under AGPL version 3 or later. Please see [the licence file](https://github.com/ppy/osu-web/blob/master/LICENCE) for more information.
```

Comment:
the figma contains both lazer and web designs, but i don't think the figma stuff has any licence specified from what i can tell

Reply:
from what I was able to find, most, if not all lazer assets are used on the web and are therefore under AGPL, too. the remaining part, hosted in ppy/osu-resources, is *mostly* [CC-BY-NC 4.0](https://github.com/ppy/osu-resources/blob/master/LICENCE.md), save for fonts

Reply:
No licence means copyright by default. Which is where we will leave things for figma. They are designs in progress and not to be considered CC or otherwise.

Reply:
this raises a few questions:
- what are the usage rules for designs of the osu! cookie, mod and modE icons, and other things featured previously in the design kit?
- same, but for WIP? 
- can Figma be used as a design source at all in this case?

my understanding is that for (1) and (2) it's "use with attribution, we don't care for as long as you mention the copyright and don't make profit off it", but that is not "fair use" either

Reply:
> * what are the usage rules for designs of the osu! cookie, mod and modE icons, and other things featured previously in the design kit and considered ready for use?

Everything in the asset kit is free for all use. They are assets provided to be used in designs or external applications. Still need to understand trademarks, which are a separate thing altogether.

> * same question, but for WIP assets — can they be used for personal things and small projects clearly not affiliated with osu!?

No

> * basically, can Figma be used as a design source at all in this case, and if yes, what should I add as a word of caution?

Figma is public only so the designs can be implemented in osu! projects. Any other use should be avoided.

Reply:
alright, what I am trying to solve is the design kit being mostly out of date (https://github.com/ppy/osu-wiki/issues/7439). if linking Figma for color/asset stuff isn't the way to go, then what is? perhaps a curated archive of some things exported from Figma, then?

Reply:
Yeah, it should be updated. But i'm not sure that will ever happen.

Reply:
alright I'm rolling this chunk back as it is more sophisticated than it looked like from the start